spoonbill/cli.py

```diff
@@ -158,6 +158,7 @@ def cli(
     ) = detect_format(filename)
     click.echo(_("Input file is {}").format(click.style(input_format, fg="green")))
     is_package = "package" in input_format
+    combine_choice = combine if combine else ""
```

Comment:
not sure why this is needed

Reply:
later in code "combine" overshedowed by other variables. 
'combine' required in order to produce correct message